### PR TITLE
[Travis] Use Selenium downloaded by Composer

### PR DESCRIPTION
**Pull request by pamil**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: etc/travis/prepare-javascript [hunk 1]
```diff
@@ -3,11 +3,7 @@
 /sbin/start-stop-daemon --start --quiet --pidfile /tmp/xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1680x1050x16
 export DISPLAY=:99
 
-# Downloading and configuring Selenium & ChromeDriver
-if [ ! -f $SYLIUS_CACHE_DIR/selenium.jar ]; then
-    curl http://selenium-release.storage.googleapis.com/2.47/selenium-server-standalone-2.47.1.jar > $SYLIUS_CACHE_DIR/selenium.jar
-fi
-
+# Downloading and configurin ChromeDriver
```
Comment (michalmarcinkowski) [inline etc/travis/prepare-javascript]: configurin?

Comment (pamil) [inline etc/travis/prepare-javascript]: I'm trying to be more laid back :)

Comment (michalmarcinkowski) [inline etc/travis/prepare-javascript]: Then :+1: :laughing:

**Comment by pjedrzejewski** [comment]
:+1:

**Pull request merged**